src/v2/view-builder/views/ov/NumberChallengePushView.js

```diff
@@ -0,0 +1,73 @@
+import { _, loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import polling from '../shared/polling';
+import { WARNING_TIMEOUT } from '../../utils/Constants';
+import ResendNumberChallengeView from './ResendNumberChallengeView';
+import NumberChallengePhoneView from './NumberChallengePhoneView';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    noButtonBar: true,
+
+    className: 'okta-verify-number-challenge',
+
+    events: {
+      'click a.resend-number-challenge': 'handleResendNumberChallenge'
+    },
+
+    handleResendNumberChallenge () {
+      this.options.appState.trigger('invokeAction', 'currentAuthenticator-resend');
+      // remove the warning
+      this.stopPoll();
```

Comment:
instead startPoll can always call `this.stopPoll` first before starting polling...

Reply:
That might not apply for when the polling start the first time.

Reply:
yeah that would be a `null` action then..